cpapp_backend/cpapp/services/oculon_client.py

```
@@ -0,0 +1,127 @@
+            
+            response = self.session.request(method, url, **kwargs)
+            
+            # Log response details for debugging
+            logger.info(f"Response status: {response.status_code}")
+            logger.debug(f"Response headers: {dict(response.headers)}")
+            
+            try:
+                response.raise_for_status()
+                response_data = response.json()
+                logger.debug(f"Response data: {response_data}")
+                return response_data
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)